stratum/hal/lib/common/utils.cc

```diff
@@ -318,34 +316,33 @@ bool ConvertTrunkMemberBlockStateToBool(const TrunkMemberBlockState& state) {
   return state == TRUNK_MEMBER_BLOCK_STATE_FORWARDING;
 }
 
-std::string MacAddressToYangString(
-    const ::google::protobuf::uint64& mac_address) {
+std::string MacAddressToYangString(const uint64& mac_address) {
   return absl::StrFormat("%x:%x:%x:%x:%x:%x", (mac_address >> 40) & 0xFF,
```

Comment:
```suggestion
  return absl::StrFormat("%2x:%2x:%2x:%2x:%2x:%2x", (mac_address >> 40) & 0xFF,
```

Reply:
It was `%02x` for zero padding in the end. Feel free to extend the new tests cases.

Reply:
Yes, sorry.